docs/lib/routes.js

```diff
@@ -42,7 +43,8 @@ const routes = (
     <Route path="/" component={UI.Layout}>
       <IndexRoute component={Home} />
       <Route path="/components/" component={Components}>
-        <IndexRedirect to="alerts/" />
```

Comment:
Let's add the alerts page back :)

Reply:
I think accordion should be the new redirect, no? If everything is alphabetic 🤔

Reply:
Ah I mean adding the alerts page back since it was deleted entirely.  The accordion page can stay as the main redirect, but there's no more alerts page on this routes list.

Reply:
Actually disregard.  Didn't realize alerts was declared below

Reply:
OH gotcha - didn't realize this `IndexRedirect` was the only link to the alerts page. 👍  Will do

Reply:
Actually - isn't it here? https://github.com/reactstrap/reactstrap/pull/2105/files#diff-68a33def9ed9e0ff643d9b783aac860a707fd6f5652fffe863dd5123bdac502dR69

Is there something else I'm missing? 😅

Reply:
Yeah sorry my bad.  Ignore my comment above - didn't notice alerts page was already there